index.html

```diff
@@ -1,54 +1,273 @@
-    rel="stylesheet"
-    />
-
-  -->
-    <link rel="stylesheet" href="style.css" />
-
-    <script
-      src="https://kit.fontawesome.com/26c38390b4.js"
-      crossorigin="anonymous"
-    ></script>
-  </head>
-  <body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Mobily First</title>
+
+  <link rel="stylesheet" href="style.css" />
+
+  <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  
     <header class="navbar">
       <a href="#" class="brand-title">My logo</a>
-      <i class="fas fa-bars"></i>
+      <img src="./images/Union.png" alt="">
     </header>
 
-    <section class="presentation">
-      <h2>I’m Alexandra. Glad to see you!</h2>
-    </section>
-
-    <article class="job">
-      <p>
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-    </article>
-    <section class="contact">
-      <p>LET’S CONNECT</p>
-    </section>
-    <section class="socialmedia">
-      <ul>
-        <li><i class="fa-brands fa-twitter"></i></li>
-        <li><i class="fa-brands fa-linkedin"></i></li>
-        <li><i class="fa-solid fa-m"></i></li>
-        <li><i class="fa-brands fa-square-github"></i></li>
-        <li><i class="fa-regular fa-hand-peace"></i></li>
-      </ul>
-    </section>
-  </body>
-</html>
+    <div class="seconback">
+      <section class="presentation">
+        
+        <h1><Strong>I’m Gonzalo. Glad to see you!</Strong></h1>
+
+        <article class="job">
+          <p>
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contactme">
+          <p>LET’S CONNECT</p>
+        </section>
+        <section class="socialmedia">
+          <ul>
+            <li class="twitter"><img src="./images/twitter.png" alt=""></li>
+            <li><img src="./images/Linkedin icon.svg" alt="">            </li>
+            <li><img class="m" src="./images/m.png" alt=""></li>  <!--Check Img Path because it not visible!-->
+            <li><img src="./images/git.png" alt=""></li>
+            <li><img src="./images/hand.png" alt=""></li>
+          </ul>
+        </section>
+      </section>
+
+      <section class="cards">
+        
+        <div class="card">
+
+          <img class="cardimg" src="./images/hischool.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
```

Comment:
- [ ] **[OPTIONAL]** Please make sure projects titles (e.g., "Tonic") uses a header tag `hx`. 
This will improve highly the accessibility of your website 👍 💯 

**Note:** This project requirement was missed by the previous reviewer so please consider implementing it.